<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="frameworkdemo.bootstrapdropdowndemo" tests="4" time="54.269" ignored="0" failures="1" timestamp="01 Apr 2020 07:09:23 GMT" skipped="0" hostname="karthik-PC" errors="0">
  <testcase name="bootstrapdropdown" time="17.091" classname="frameworkdemo.bootstrapdropdowndemo"/>
  <testcase name="bootstrapdropdown2" time="10.916" classname="frameworkdemo.bootstrapdropdowndemo">
    <failure type="java.lang.AssertionError" message="Lists differ at element [0]: Angular != Web Development expected [Angular] but found [Web Development]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [0]: Angular != Web Development expected [Angular] but found [Web Development]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:568)
	at org.testng.Assert.assertEquals(Assert.java:533)
	at frameworkdemo.bootstrapdropdowndemo.bootstrapdropdown2(bootstrapdropdowndemo.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- bootstrapdropdown2 -->
  <testcase name="bootstrapdropdown3" time="12.926" classname="frameworkdemo.bootstrapdropdowndemo"/>
  <testcase name="bootstrapdropdown1" time="13.336" classname="frameworkdemo.bootstrapdropdowndemo"/>
</testsuite> <!-- frameworkdemo.bootstrapdropdowndemo -->
